etc/kayobe/environments/stackhpc-baremetal/kolla.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+kolla_enable_ironic: true
```

Comment:
kolla_enable_ironic_inspector: false needs to go here.

@assumptionsandg there are other ones too I think? like dnsmasq?